llvm/test/CodeGen/AMDGPU/code-size-estimate.ll

```diff
@@ -1,6 +1,7 @@
 ; RUN: llc -march=amdgcn -mcpu=gfx900 -show-mc-encoding < %s | FileCheck -check-prefixes=CHECK,GFX9 %s
 ; RUN: llc -march=amdgcn -mcpu=gfx1030 -show-mc-encoding < %s | FileCheck -check-prefixes=CHECK,GFX10 %s
 ; RUN: llc -march=amdgcn -mcpu=gfx1100 -show-mc-encoding < %s | FileCheck -check-prefixes=CHECK,GFX11 %s
```

Comment:
You missed adding the GFX1100 prefix to this RUN line

Reply:
https://github.com/llvm/llvm-project/commit/a657deb42ebdfb5e72d0823001485a97ee65a2c6

Reply:
Thanks!